src/CommandCompleter.cpp

```diff
@@ -55,6 +55,8 @@ CommandCompleter::data(const QModelIndex &index, int role) const
                 return QString("/unban @");
             case Redact:
                 return QString("/redact ");
+            case MyRoomnick:
```

Comment:
You are missing the MyRoomnick case in the other switch statements it seems?

Reply:
You're right! Code updated :)